redbot/cogs/mod/kickban.py

```diff
@@ -254,6 +283,12 @@ async def _check_tempban_expirations(self) -> None:
                 if await self._check_guild_tempban_expirations(guild, guild_tempbans):
                     await self.config.guild(guild).current_tempbans.set(guild_tempbans)
 
+    def transform_message(self, message, objects):
+        results = re.findall(r"{([^}]+)\}", message)
```

Comment:
This may work better as a compiled regex string?

Reply:
Considering it was only ran on a mod action, I didn't think it was really necessary to do so but can change it otherwise. (it's also not compiled in custom com)